server/controllers/conversation.js

```diff
@@ -0,0 +1,48 @@
+const asyncHandler = require("express-async-handler");
+const Conversation = require("../models/Conversation");
+const Profile = require("../models/Profile");
+
+//For scenario where ownerId will contact other user Via profile
+exports.createConversationViaProfile = asyncHandler(async (req, res, next) => {
+  const senderId = req.user.id;
+  const { profileId } = req.body;
```

Comment:
You can validate `profileId` first so you can return a bad request status if the client did not provide it

Reply:
Validation added